nodejs/eks/cmd/provider/cni.ts

```diff
@@ -167,7 +167,7 @@ function applyVpcCniYaml(args: VpcCniInputs) {
     // Check to ensure that a compatible kubectl is installed, as we'll need it in order to deploy k8s resources below.
     assertCompatibleKubectlVersionExists();
 
-    const yamlPath = path.join(__dirname, "..", "..", "cni", "aws-k8s-cni.yaml");
+    const yamlPath = path.join(__dirname, "../../cni/aws-k8s-cni.yaml");
```

Comment:
This feels like it defeats the point of using `join`, but pkg seems to want a specifically two-argument `join` in order to pick it up as an asset (same for the dashboard below).

https://github.com/vercel/pkg#detecting-assets-in-source-code

Reply:
Hmm, we use `path.join` to make this cross-platform. Using literal `/` won't work on Windows. Although, I guess this doesn't matter because this won't actually be run on Windows because it will be run at build-time only.